docs/start/install.rst

```diff
@@ -247,7 +386,143 @@ know how to resolve them. If none of these help you, please ask a question on
   The most likely cause of this error is a misconfigured ``PYTHONPATH``
   environment variable. Unset that variable and try again.
 
+``ModuleNotFoundError: No module named 'gendoc'``
+  Check your python version with ``python --version``. if you are running 
+  ``python3.10``, you will need to :ref:`install the latest dev version<Install Dev Version>`
+  or :ref:`build from source<Install from Source>` or revert to ``python3.9``.
+
+  - This issue is resolved by `Include gendoc file into the source distribution (#3151)`_ is
+    currently available in the :ref:`latest development version<Install Dev Version>`.
+
+``ImportError: DLL load failed while importing _datatable: The specified module could not be found.``
+  Check your python version with ``python --version``. If you are running 
+  ``python3.11``, you will need to revert to ``python3.9``. Alternatively, you may 
+  use ``python3.10`` and :ref:`install the latest dev version<Install Dev Version>`
+  or :ref:`build from source<Install from Source>`.
+
+  - This issue is an active feature request to `add support for python 3.11 (#3374)`_
+    and will be included in `Release 1.1.0`_ of datatable. 
+
+.. _PVE:
+
+Appendix: Creating a Python Virtual Environment
```

Comment:
If you do think this may be useful for datatable users, may be just put a couple of external links to venv documentation? Like installing venv on different platforms.